backend/data/static/create_master_files.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd
+import os
+from functools import reduce
+
+# Set up directory for the os
+directory = "cleaned/"
+
+# Initialise some values for the loop
+la = {}
+i = 0
+
+# Read in all the dataframes to a dictionary
+for filename in os.listdir(directory):
+    if filename.startswith("LA_"):
```

Comment:
this `'LA_'` should be a parameter of this function - pres. called `prefix` or `filename_prefix` whose default value is `"LA_"` so:

```
def collect_las(filename_prefix="LA_"):
   ...
```